src/platform/datapath_raw.h

```diff
@@ -67,6 +69,9 @@ typedef struct CXPLAT_DATAPATH {
     BOOLEAN Uninitialized : 1;
     BOOLEAN Freed : 1;
 #endif
+#ifdef QUIC_USE_RAW_DATAPATH
+    BOOLEAN UseTcp;
+#endif
```

Comment:
We're in datapath_**raw**.h, shouldn't `QUIC_USE_RAW_DATAPATH` always be defined?
```suggestion
    BOOLEAN UseTcp;
```